docs/concept-exercises.md

```diff
@@ -51,30 +51,51 @@ An exercise has the following files. In the browser they will show at the releva
+**Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
-As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes". Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
+- The information provided should give the student just enough context to figure out the solution themselves. 
+- It should not give the student any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
+- Use the proper technical terms so that the student can easily search for more information. 
+- If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+
+As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes", tell the users how to create a string, and explain that a string has methods that can be used to manipulate it. Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
 
 See the C# floating-point-numbers exercise's [introduction.md file][csharp-docs-introduction.md] for an example.
 
 ### `.docs/instructions.md`
 
-This file contains instructions for the exercise. It should explicitly explain what the student needs to do (define a method with the signature `X(...)` that takes an A and returns a Z), and provide at least one example usage of that function. If there are multiple tasks within the exercise, it should provide an example of each.
+**Purpose:** Provide instructions for the exercise.
+
+This file is split into two parts. 
+1. The first part explains the "story" or "theme" of the exercise. It should generally contain no code samples.
+2. The second part provides clear instructions of what a student needs to do, providing tasks.
+
+Each task must conform to the following standard:
+- Start with a third-level heading starting with a number (.e.g `### 1.`, `### 2.`)
+- Describe which function/method the student needs to define/implement (e.g. `Implement method X(...)` that takes an A and returns a Z),
+- Provide an example usage of that function. These examples should be different examples to those given in the tests.
 
-See the C# floating-point-numbers exercise's [instructions.md file][csharp-docs-instructions.md] for an example.
+Avoid any controversial topics in the stories - be empathetic to triggers that people might have - there's normally a less controversial or risque variant of a story that can be used.
```

Comment:
```suggestion
Avoid any topics in the stories that could be misconstrued or cause offence - be empathetic to triggers that people might have - there's normally a less controversial or risque variant of a story that can be used.
```

Reply:
This still has the same issues that I outlined in my comment above. I wrote it at the same time you added this suggestion :D

Reply:
Perhaps instead we can turn this into a _positive_? 

Instead of **avoid x y z** which requires interpretation of the writer, we could do something like **some topics might be misconstrued or cause offence. When in doubt if this will be the case, ping @exercism/xyz in your PR**